lambda/src/test/resources/Migrations/GuardianWeekly2025/6801-Discount/account.json

```
@@ -13,8 +13,8 @@
     "purchaseOrderNumber": null,
     "customerServiceRepName": null,
     "ScrubbedOn__c": null,
-    "IdentityId__c": "IDENTITY_ID__c",
-    "sfContactId__c": "sfContactId__c",
+    "IdentityId__c": "100825957",
+    "sfContactId__c": "0030J00001uyVrKQAU",
```

Avoid having real customer data (Identity IDs in particular are considered PII)

```suggestion
    "IdentityId__c": "123456",
    "sfContactId__c": "654321",
```